chart/kubeapps/templates/kubeappsapis/service.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.featureFlags.kubeappsAPIsServer }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "kubeapps.kubeappsapis.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: kubeappsapis
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" . ) | nindent 4 }}
+    {{- end }}
+  {{- if or .Values.kubeappsapis.service.annotations .Values.commonAnnotations }}
+  annotations:
+    {{- if .Values.kubeappsapis.service.annotations }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.kubeappsapis.service.annotations "context" $) | nindent 4 }}
+    {{- end }}
+    {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- end }}
+spec:
+  type: ClusterIP
+  ports:
+    - port: {{ .Values.kubeappsapis.service.port }}
+      targetPort: http
+      protocol: TCP
+      name: http
```

Comment:
What about the gRPC port? Are we just serving the HTTP API here?

Reply:
They're both on the same port. Wasn't sure whether to call it something else, like `grpc_http`. What do you think?

Reply:
Oops, that's true... that's the key of the multiplexing indeed :P
But, yep, I'd rather change it to `grpc_http` to avoid future misunderstandings.